src/runtime_src/core/edge/drm/zocl/zocl_kds.c

```diff
@@ -1165,6 +1165,9 @@ zocl_reset(struct drm_zocl_dev *zdev, const char *buf, size_t count)
 	mutex_lock(&kds->lock);
         /* Find out number of active client and send a signal to it. */
         list_for_each_entry_safe(client, tmp_client, &kds->clients, link) {
+                if (pid_nr(client->pid) == pid_nr(task_tgid(current)))
```

Comment:
client->pid is "get_pid(task_pid(current))" in kds_core.c. Do we really want to compare with task_tgid()?

Reply:
Actually this checks is to ignore xbutil  as a client. While we are killing the PID of all the active client. We should not kill the xbutil itself, which is requested for reset.